### ISSUE TITLE
Full testing

### ISSUE THREAD

**Issue by matteot11** [gurkirt/AMTNet.pytorch#3]
Hi, thank you for the great codebase.

I was wondering how to evaluate trained models on the full split-1 test set of ucf24, and not only on the subset with 15k images.
Is it enough to set the fulltest flag to True, in the make_lists function defined in data/shared.py?
Another question: do you have any suggestions to compute the video-mAP (I can find the frame-mAP evaluation code but not the video-mAP one)?
Thank you in advance.

Matteo

**Comment by gurkirt** [comment]
Current evaluate.py does evaluate the entire test set.

You will need to use Matlab code for video-mAP.

**Comment by matteot11** [comment]
Hi, thanks for the reply.

I successfully run the evaluate.py script, but it evaluates on 11752 frames for ucf24, in its current form. During training, there is an evaluation step every 10000 iterations on the same number of frames (11752) and in the README I see:
"During training checkpoint is saved every 10K iteration also log it's frame-level frame-mean-ap on **a subset of 15k test images**." 

If I initialize the ActionDetection dataset with "full_test=True", the number of frames during evaluation becomes 79445.

Matteo
